tasks/Functions-consumer/performManualUpkeep.js

```diff
@@ -16,22 +16,42 @@ task("functions-perform-upkeep", "Manually call performUpkeep in an Automation c
     // Call performUpkeep
     const performData = taskArgs.data ?? []
 
+    const autoConsumerContractFactory = await ethers.getContractFactory("AutomatedFunctionsConsumer")
+    const autoConsumerContract = await autoConsumerContractFactory.attach(taskArgs.contract)
+
+    const checkUpkeep = await autoConsumerContract.checkUpkeep(performData)
+    if (!checkUpkeep.upkeepNeeded) {
+      console.log("\ncheckUpkeep returned false. Upkeep was not performed.")
+      return
```

Comment:
This might be a little annoying if somebody wants to force a performUpkeep but now needs to wait for s_updateInterval to pass. How is this command expected to be used? Maybe put calling checkUpkeep under a flag?

Reply:
Noted. I can just remove this check and allow `performUpkeep` to revert if `checkUpkeep` returns false.